examples/cornerstoneDICOMSR/index.html

```diff
@@ -88,9 +88,8 @@ <h1>
         <script src="https://unpkg.com/cornerstone-wado-image-loader@3.0.2/dist/cornerstoneWADOImageLoader.min.js"></script>
         <script src="https://unpkg.com/hammerjs/hammer.min.js"></script>
 
-        <script src="/js/initCornerstone.js"></script>
-
-        <script src="/js/dcmjs.js"></script>
+        <script src="../js/initCornerstone.js"></script>
+        <script src="../js/dcmjs.js"></script>
```

Comment:
These paths are different between local testing and deployed examples.  Are you sure they should be changed as part of this PR?

Reply:
So I was the one who initially changed these but I wasn't sure if they should be changed from /js/ to ../js/. I did so to make it work locally and then kept it when I realized that the deploy preview (https://deploy-preview-187--dcmjs.netlify.app/) worked with the change to ../js/ as well. But I did note that the currently deployed previews work great with /js/ rather than ../js/.

Reply:
I think @gangaramstyle is right, if you go to the current deployed example of the current master branch (e.g.: https://dcmjs.netlify.app/displaysegmentation/) are broken and they return this error:

Uncaught ReferenceError: cornerstoneWADOImageLoader is not defined
    at initCornerstone.js:40

while it works in https://deploy-preview-187--dcmjs.netlify.app/displaysegmentation/ (i.e.: importing by ../js/).

Reply:
Update: I can't reproduce this, probably I had to clear my web browers cache. I will delete reverse teh commit modifying the import.